### DISCUSSION TITLE
font setup for ubuntu server console

### DISCUSSION THREAD

**Discussion by lukaszgiez** [JanDeDobbeleer/oh-my-posh#3186] [Troubleshoot]
Hello,
Could you help me with setting up my Ubuntu server console to use the Meslo font please?
My shell is bash.
Currently, I installed oh-my-posh and I set up the terminal in the gui environment; this works just fine.
However, when I log in to the server remotely (via putty), I do not get the correct font in the terminal (pts). Instead, I get the squares and symbols, where icons should be.
Hope someone can help! :)

**Comment by JanDeDobbeleer** [comment]
@lukaszgiez you need to setup the same font in the terminal where you ssh from.

**Reply by lukaszgiez** [reply]
Thank you!
This has worked. I needed to define the font in putty for the default session and for all saved sessions.

Im wondering now that this creates a problem if more people log in as the same user to the machine, and some of them do not know they need to have this font. If I switch on oh-my-posh in .bashrc, then all these people will see faulty prompts.
Is there a workaround short of setting up discrete users?

**Reply by JanDeDobbeleer** [reply]
you could print a default welcome message but that's always a client issue

**Reply by lukaszgiez** [reply]
ok, thank you kindly for the super-fast replies! :)
